SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -725,6 +725,13 @@ int64 USpatialActorChannel::ReplicateActor()
 			const VirtualWorkerId NewAuthVirtualWorkerId = NetDriver->LoadBalanceStrategy->WhoShouldHaveAuthority(*Actor);
 			if (NewAuthVirtualWorkerId != SpatialConstants::INVALID_VIRTUAL_WORKER_ID)
 			{
+				// Update Actor SpawnData location for when new worker receives authority to prevent hysteresis.
+				SpatialGDK::SpawnData* ActorSpawnData = NetDriver->StaticComponentView->GetComponentData<SpatialGDK::SpawnData>(EntityId);
+				check(ActorSpawnData != nullptr);
+				ActorSpawnData->Location = Actor->GetActorLocation();
```

Comment:
Should we update the entire SpawnData as well ? 
Also, could we wrap this operation into a function ?

Reply:
Haha, this is what my first commit on this branch did. However, this caused issues. When a player crossed the boundary and stopped moving, the velocity value would continue being applied and the player would just keep sliding. Only the position was required for fixing the load balancing issue and in my testing this was fine for the player crossing boundaries and projectiles. 

I feel like a more complete fix is ensuring the correct replicated movement data is applied before the first ReplicateActor call on the new worker. However, I think this would be a little more of an involved investigation and doing this mitigates a prominent issue as is.